### PR TITLE
Make fileupload question in its own block

### PR DESCRIPTION
**Pull request by hmhsu**
### Description
Ensure admin can only add fileupload question in its own block.

### Checklist
- [ ] Created tests which fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

### CODE REVIEW

File: universal-application-tool-0.0.1/app/views/components/QuestionBank.java [hunk 1]
```diff
@@ -160,14 +159,13 @@ private ContainerTag renderQuestionDefinition(QuestionDefinition definition) {
    * <p>Questions that are filtered out:
    *
    * <ul>
-   *   <li>If there is at least one question in the current block, all enumerator questions
+   *   <li>If there is at least one question in the current block, all single-block questions
    *   <li>If this is a repeated block only show the appropriate repeated questions
    *   <li>Questions already in the program
```
Comment (kendrickt) [inline universal-application-tool-0.0.1/app/views/components/QuestionBank.java:164]: (blocking) Is this missing a case?

`If there is single block question, all questions are filtered.`

File: universal-application-tool-0.0.1/app/views/components/QuestionBank.java [hunk 2]
```diff
@@ -176,6 +174,11 @@ private ContainerTag renderQuestionDefinition(QuestionDefinition definition) {
     return questions.stream().filter(filter).collect(ImmutableList.toImmutableList());
   }
 
+  /** If a block already contains a single-block question, no more questions can be added. */
+  private boolean noMoreQuestionsAllowed() {
```
Comment (kendrickt) [inline universal-application-tool-0.0.1/app/views/components/QuestionBank.java:178]: (blocking) rename `noMoreQuestionsAllowed` to `containsSingleBlockQuestion` to keep re-using the new "single block question" term.

File: browser-test/src/ti_group.test.ts [hunk 3]
```diff
@@ -1,19 +1,19 @@
-import { startSession, loginAsAdmin, endSession, AdminTIGroups } from './support'
```
Comment (kendrickt) [inline browser-test/src/ti_group.test.ts:1]: Is this a carriage return diff?

**Review by kendrickt — CHANGES_REQUESTED** [review]
(blocking) I think there are a couple files that are just whitespace changes. Not sure what's going on but can you not include them in this PR? If we need to push those changes it should be in a separate PR.

(blocking maybe?) do we need tests for this change?

File: browser-test/src/ti_group.test.ts (hunk 3 above)
Comment (hmhsu) [reply browser-test/src/ti_group.test.ts:1]: Yes, they are formatting changes only.

File: universal-application-tool-0.0.1/app/views/components/QuestionBank.java (hunk 1 above)
Comment (hmhsu) [reply universal-application-tool-0.0.1/app/views/components/QuestionBank.java]: Done.

File: universal-application-tool-0.0.1/app/views/components/QuestionBank.java (hunk 2 above)
Comment (hmhsu) [reply universal-application-tool-0.0.1/app/views/components/QuestionBank.java]: Done.

**Comment by hmhsu** [comment]
> I think there are a couple files that are just whitespace changes. Not sure what's going on but can you not include them in this PR? If we need to push those changes it should be in a separate PR.
Removed

> Do we need tests for this change?
I think the existing test for enumerator question/block is sufficient as they use exactly the same logic.

**Review by kendrickt — APPROVED** [review]
lgtm

**Comment by hmhsu** [comment]
Linking - this works toward #404.

**Pull request merged**